umiCollapse.wdl

```diff
@@ -330,6 +332,8 @@ task bamSplitDeduplication {
         modules: "Required environment modules"
         memory: "Memory allocated for this job"
         timeout: "Time in hours before task timeout"
+        method: "What method to use to identify group of reads with the same (or similar) UMI(s)?"
+        editDistanceThreshold: "Parametr for the adjacency and cluster methods, the threshold for the edit distance to connect two UMIs in the network."
     }
 
     command <<<
```

Comment:
A good practice to use **set -euo pipefail** in command block if there are more than one commands.